src/api/video/video.controller.js

```diff
@@ -0,0 +1,41 @@
+const Videos = require('./video.model');
```

Comment:
The convention is to use singular `Video` for the model name.

Reply:
Totally agree... yeah I was kept 2nd guessing myself whether or not I should include the "s" in diff situations